hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/resolver/SimpleEndpointAddressResolver.java

```diff
@@ -0,0 +1,6 @@
+package pl.allegro.tech.hermes.consumers.consumer.sender.resolver;
+
+
+public class SimpleEndpointAddressResolver implements EndpointAddressResolver {
```

Comment:
This is the most basic form of address resolver which takes as an input only actual address. This is especially useful for test scenarios and batch delivery.